packages/blockchain-link/package.json

```diff
@@ -76,6 +76,7 @@
         "worker-loader": "^3.0.8"
     },
     "dependencies": {
+        "@everstake/wallet-sdk": "^1.0.0",
```

Comment:
ah great, please remove hardcoded contract addresses for ETH

Reply:
also changed here https://github.com/trezor/trezor-suite/pull/15409/commits/8143354243fc6c1a337c480b5852f01a90121c61